install.js

```diff
@@ -414,7 +422,22 @@ var steps = [
   },
   // all done
   function finalize (next) {
-    console.log("Installation complete.\nRun the command 'node server' (or 'foreman start' if using heroku toolbelt) to start your instance.");
+    var passedArgs = process.argv.slice(2);
+
+    for (var i = 2; i < passedArgs.length; i++) {
+      if (passedArgs[i] == undefined) {
+        isVagrantInstallation = false;
+        break;
+      } else {
+        isVagrantInstallation = true;
+      }
+    }
+
+    if (isVagrantInstallation) {
```

Comment:
@brian-learningpool see most recent commit, I have taken yours and Dennis' points on board.